.github/workflows/csim&rvfi.yml

```diff
@@ -0,0 +1,24 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: [ubuntu-18.04]
+    steps:
+    - name: Install opam2
+      run: |
+        sudo add-apt-repository -y ppa:avsm/ppa
+        sudo apt install -y opam zlib1g-dev pkg-config libgmp-dev z3
+    - name: Init opam
+      run: opam init --disable-sandboxing -y
+    - name: Install sail
+      run: opam install -y sail
+    - name: Check out repository code
+      uses: actions/checkout@HEAD
+      with:
+        submodules: true
+    - name: Build RV32 simulators
+      run: eval $(opam env) && make ARCH=RV32 -j2 csim rvfi
+    - name: Build RV64 simulators
+      run: eval $(opam env) && make ARCH=RV64 -j2 csim rvfi
```

Comment:
I dont think changing workflow script does any good in this instance, if there is an issue with the CI script, that should be addressed in a different PR, I advise you to delete this and keep the original one.